openage/config.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2014-2016 the openage authors. See copying.md for legal info.
+
+"""
+Project configuration
+"""
+
+import site
+
+
+DEVMODE = not any([__file__.startswith(path) for path in site.getsitepackages()])
```

Comment:
Cool idea :) We should `normpath()` the `path` and the `__file__` to have a safe prefix check.